moveit_ros/benchmarks/CMakeLists.txt

```diff
@@ -1,39 +1,47 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(moveit_ros_benchmarks)
 
+set(MOVEIT_LIB_NAME moveit_ros_benchmarks)
+
 add_compile_options(-std=c++11)
 
 if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
-find_package(Boost REQUIRED COMPONENTS date_time filesystem program_options system thread)
+find_package(Boost REQUIRED filesystem)
 
-find_package(catkin REQUIRED COMPONENTS
+find_package(catkin COMPONENTS
   moveit_ros_planning
   moveit_ros_warehouse
-  tf
-  eigen_conversions
   roscpp
-  rosconsole
+  REQUIRED
 )
 
 catkin_package(
-  INCLUDE_DIRS
-    benchmarks/include
-  LIBRARIES
-    moveit_benchmark_execution
+  LIBRARIES ${MOVEIT_LIB_NAME}
   CATKIN_DEPENDS
     moveit_ros_planning
     moveit_ros_warehouse
+    roscpp
+  DEPENDS
+  INCLUDE_DIRS ${CMAKE_CURRENT_LIST_DIR}/include
 )
 
-include_directories(benchmarks/include)
+include_directories(${catkin_INCLUDE_DIRS} ${CMAKE_CURRENT_LIST_DIR}/include)
+link_directories(${catkin_LIBRARY_DIRS})
+
+add_library(${MOVEIT_LIB_NAME} src/BenchmarkOptions.cpp
+                               src/BenchmarkExecutor.cpp)
+target_link_libraries(${MOVEIT_LIB_NAME} ${catkin_LIBRARIES} ${Boost_LIBRARIES})
 
-include_directories(${catkin_INCLUDE_DIRS})
-include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
+add_executable(moveit_run_benchmark src/RunBenchmark.cpp)
+target_link_libraries(moveit_run_benchmark ${MOVEIT_LIB_NAME} ${catkin_LIBRARIES} ${Boost_LIBRARIES})
 
-link_directories(${Boost_LIBRARY_DIRS})
-link_directories(${catkin_LIBRARY_DIRS})
+install(
+  TARGETS
+    ${MOVEIT_LIB_NAME} moveit_run_benchmark
+  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
 
-add_subdirectory(benchmarks)
+install(DIRECTORY include/ DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION})
```

Comment:
please respect http://moveit.ros.org/documentation/contributing/code.html .
This probably installs the headers to the wrong folder.